src/PendingCalls/TestCall.php

```diff
@@ -172,6 +173,17 @@ public function skip(Closure|bool|string $conditionOrMessage = true, string $mes
         return $this;
     }
 
+    /**
+     * Skips the current test when no set group.
+     */
+    public function skipUnlessGroup(string $group, string $message = ''): TestCall
```

Comment:
I suggest the name of this method could be **skipUnlessGroupIs**

Reply:
ls at the end of the name can confuse the person and is "I" or "L"